shared/media-accelerator/develop/multi-channel-streaming/project-implementation/android.mdx

```diff
@@ -0,0 +1,340 @@
+<PlatformWrapper platform="android">
+
+This section shows you how to implement the following multi-streaming methods:
+
+- [Stream to multiple channels](#stream-to-multiple-channels)
+
+- [Push multiple streams to a single channel](#push-multiple-streams-to-a-single-channel)
+
+To implement either of the two multi-streaming methods, [update the UI](#implement-the-user-interface) and then follow the step-by-step procedure in the relevant section.
+
+### Implement the user interface
+
+ You add two buttons to the UI to enable users to start and stop sending two streams. 
+ 
+ 1. In `/app/res/layout/activity_main.xml`, add the following code before `</RelativeLayout>`:
+
+    ```xml
+    <Button
+        android:id="@+id/Stream1Button1"
+        android:layout_margin="5dp"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_below="@id/JoinButton"
+        android:onClick="stream1"
```

Comment:
Global thought, not stream1 or stream2, call then audioStream and videoStream or something that describes what they are please.

Reply:
The user interface is common to both methods of multi-streaming that we present in this doc. 
If I use specific names, I will need to repeat the UI section for each multi-streaming method.
In the first method, we send an audio stream and a video stream over two channels, in the second we have two video streams on the same channel.